rust/ql/lib/codeql/rust/internal/ConceptsShared.qll

```diff
@@ -30,7 +30,7 @@ module Cryptography {
   class PasswordHashingAlgorithm = CryptoAlgorithms::PasswordHashingAlgorithm;
 
   /**
-   * A data-flow node that is an application of a cryptographic algorithm. For example,
+   * A data flow node that is an application of a cryptographic algorithm. For example,
```

Comment:
Don't forget to change the sync'd copies as well.

Reply:
Are you happy that we do that?  I didn't want to interfere with other languages unless there's some level of agreement.